jsapp/js/account/useProducts.hook.ts

```diff
@@ -0,0 +1,38 @@
+import {createContext, useEffect, useState} from 'react';
+import type {Product} from 'js/account/stripe.types';
+import {getProducts} from 'js/account/stripe.api';
+import useWhenStripeIsEnabled from 'js/hooks/useWhenStripeIsEnabled.hook';
+
+export interface ProductsState {
+  products: Product[];
+  isLoaded: boolean;
```

Comment:
Do we use this enough to abstract "isLoading" into a base interface? Once we add refresh logic, we'll also likely want a isInitialLoad or whatever chosen phrase.